docs/docs/theory_2.md

```diff
@@ -44,10 +44,109 @@ $$
+			+\frac{1}{2d}\mathrm{Tr}\Bigg[(\sigma_\mu^{\hat n} \otimes \mathbb{1}_{d}) \sigma_\mu^{\hat n}(t)\Bigg].
+$$
+
+Note that ass the trace of a Pauli spin matrix is **always** zero, only the second term is non-zero. Hence the muon polarisation can be written 
+as (after re-defining $\sigma_\mu^{\hat n}$ to include the kronecker product with the identity matrix of the other spins)
+
+$$
+P^{\hat n}_\mu(t) = \frac{1}{2d}\mathrm{Tr}\Bigg[\sigma_\mu^{\hat n}(0) \sigma_\mu^{\hat n}(t)\Bigg].
+$$
+
+Now replacing the trace with $\sum_\alpha \langle \alpha| ... | \alpha \rangle$, where $| \alpha \rangle$ is a complete set 
+of orthonormal eigenstates we obtain
+
+$$
+P^{\hat n}_\mu(t) = \frac{1}{2d}\sum_\alpha \langle \alpha | \Bigg[\sigma_\mu^{\hat n}(0) \sigma_\mu^{\hat n}(t)\Bigg] | \alpha \rangle.
+$$
+
+Explicitly writing out the time dependance, we get
+
+$$
+P^{\hat n}_\mu(t) = \frac{1}{2d}\sum_{\alpha, \beta, \gamma} \langle \alpha | \sigma_\mu^{\hat n}(0) |\beta \rangle e^{iE_\beta t} \langle \beta | \sigma_\mu^{\hat n}(0)
+|\gamma \rangle e^{-iE_\gamma t} \langle \gamma | \alpha \rangle,
+$$
+
+and as $\langle \gamma | \alpha \rangle = \delta_{\gamma, \alpha}$, this simplifies to
+
+$$
+P^{\hat n}_\mu(t) = \frac{1}{2d}\sum_{\alpha, \beta} \langle \alpha | \sigma_\mu^{\hat n}(0) |\beta \rangle e^{iE_\beta t} \langle \beta | \sigma_\mu^{\hat n}(0)
+|\alpha \rangle e^{-iE_\alpha t},
+$$
+
+so that
+
+$$
+P^{\hat n}_\mu(t) = \frac{1}{2d}\sum_{\alpha, \beta} \Big| \langle \alpha | \sigma_\mu^{\hat n}(0) |\beta \rangle \Big|^2 e^{i(E_\beta-E_\alpha) t}.
+$$
+
+Then, we can separate these terms into
+
+$$
+\begin{aligned}
+P^{\hat n}_\mu(t) & = \frac{1}{2d}\sum_{\alpha = \beta} \Big| \langle \alpha | \sigma_\mu^{\hat n}(0) |\beta \rangle \Big|^2 e^{i(E_\beta-E_\alpha) t} \\
+& + \frac{1}{2d}\sum_{\alpha < \beta} \Big| \langle \alpha | \sigma_\mu^{\hat n}(0) |\beta \rangle \Big|^2 e^{i(E_\beta-E_\alpha) t} + \frac{1}{2d}\sum_{\alpha > \beta} \Big| \langle \alpha | \sigma_\mu^{\hat n}(0) |\beta \rangle \Big|^2 e^{i(E_\beta-E_\alpha) t}.
+\end{aligned}
+$$
+
+Now by swapping $\alpha$ and $\beta$ in the last term, we see that it is the same as the second term apart from a sign in the exponential, so they may combined to give
+
+$$
+P^{\hat n}_\mu(t) = \frac{1}{2d}\sum_{\alpha = \beta} \Big| \langle \alpha | \sigma_\mu^{\hat n}(0) |\beta \rangle \Big|^2 e^{i(E_\beta-E_\alpha) t} + \frac{1}{2d}\sum_{\alpha < \beta} \Big| \langle \alpha | \sigma_\mu^{\hat n}(0) |\beta \rangle \Big|^2 [e^{i(E_\beta-E_\alpha) t} + e^{-i(E_\beta-E_\alpha) t}].
+$$
+
+Finally, by expressing the exponentials in terms of $\sin$ and $\cos$, we may simplify the expression to
+
+$$
+P^{\hat n}_\mu(t) = \frac{1}{2d}\sum_{\alpha = \beta}\Big|\langle\alpha|\sigma_{\mu}^{\hat{n}}|\beta\rangle\Big|^2 + \frac{1}{d}\sum_{\alpha > \beta} \Big| \langle \alpha | \sigma_\mu^{\hat n}(0) |\beta \rangle \Big|^2 \cos [(E_\beta-E_\alpha) t].
```

Comment:
```suggestion
P^{\hat n}_\mu(t) = \frac{1}{2d}\sum_{\alpha = \beta}\Big|\langle\alpha|\sigma_{\mu}^{\hat{n}}|\beta\rangle\Big|^2 + \frac{1}{d}\sum_{\alpha < \beta} \Big| \langle \alpha | \sigma_\mu^{\hat n}(0) |\beta \rangle \Big|^2 \cos [(E_\beta-E_\alpha) t].
```
I think the changes here are really great, and I find it much easier to follow. One tiny point is that on line 137 the sum has `\alpha < \beta`, but then this line still uses `\alpha > \beta`. Given everything in the sum has arbitrary labels and is symmetric w.r.t swapping them this isn't inaccurate but I think it'd be nicer to have it match.

Reply:
Yes I missed that, thanks!